src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+                className("label-checkbox"));
+        assertEquals(checkButtons.size(), 4);
+        List<WebElement> radioButtons = driver.findElements(By.
+                className("label-radio"));
+        assertEquals(radioButtons.size(), 4);
+        WebElement colorDropDown = driver.findElement(By.
+                className("colors"));
+        assertTrue(colorDropDown.isDisplayed());
+        WebElement defaultButton = driver.findElement(By.
+                xpath("//button[@name='Default Button']"));
+        assertTrue(defaultButton.isDisplayed());
+        defaultButton = driver.findElement(By.
+                xpath("//input[@class='uui-button']"));
+        assertTrue(defaultButton.isDisplayed());
+
+        // 9. Assert that there is Right Section
+        // There we check if the right section is displayed
+        assertTrue(driver.findElement(By.
+                xpath("//div[@name='navigation-sidebar']")).isDisplayed());
+
+        // 10. Assert that there is Left Section
+        // There we check if the left section is displayed
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if left section doesn't exist
+        // (we'll not use them in the further steps)
+        sa.assertTrue(driver.findElement(By.
+                xpath("//div[@name='log-sidebar']")).isDisplayed());
+
+        // 11. Select checkboxes
+        // There we should click on checkboxes with labels
+        // "Water" and "Wind" and check if they are checked
+        driver.findElement(By.
+                xpath("//div[@class='main-content']//div[2]//label[1]")).
+                click();
+        WebElement waterCheckBox = driver.findElement(By.
+                xpath("//div//div//div//div[2]//label[1]//input[1]"));
+        assertTrue(waterCheckBox.isSelected());
+        driver.findElement(By.
+                xpath("//div[@class='main-content']//div[2]//label[3]")).
+                click();
+        WebElement windCheckBox = driver.findElement(By.
+                xpath("//div//div//div//div[2]//label[3]//input[1]"));
+        assertTrue(windCheckBox.isSelected());
+
+        // 12. Assert that for each checkbox there is
+        // an individual log row and value is corresponded
+        // to the status of checkbox.
+        // There we get the panel with informations about
+        // checkings and selections and get the number of
+        // rows here. We should have 2 rows. After that we
+        // check if the panel with information really contains
+        // such infos as "Water" and "Wind" and the labels "true" here
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if info about these checkboxes doesn't
+        // display (we'll not use it in the further steps)
+        WebElement actualLogRowsWebElement = driver.findElement(By.
+                className("info-panel-section"));
+        List<String> actualLogRowsValues = Arrays.
```

Comment:
1. Creation of the List could be simplified
2. Using split("\n") not good idea(

Reply:
Solved: I create list of these items from log row web element and checked its size